Assets/Scripts/ArrowTrigger.cs

```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+
+[RequireComponent(typeof(Collider))]
+public class ArrowTrigger : MonoBehaviour
+{
+    public UnityEvent triggerEnter;
+    public UnityEvent triggerExit;
+
+    private void Reset()
+    {
+        GetComponent<Collider>().isTrigger = true;
```

Comment:
What is the point of this? Is the collider normally not a trigger, then it is set to one when it is reset? Couldn't you just click isTrigger on the inspector?

Reply:
yeah, i agree, the person who creates the level can set that themselves. if an object has a collider and a trigger, getcomponent might select the collider and the object would have two triggers